test/Encoding.test.js

```diff
@@ -1,15 +1,11 @@
-'use strict';
-
-var expect = require('chai').expect;
-
-var Encoding = require('../src/Encoding').Encoding;
-
+var chai_1 = require('chai');
```

Comment:
So I guess we check in both ts, js because we haven't migrated test to run on TS?